fhir-server-test/src/test/java/com/ibm/fhir/server/test/bulkdata/ImportOperationAzureTest.java

```diff
@@ -0,0 +1,360 @@
+        target = target.path(path);
+        return target.request(mimeType).header("X-FHIR-TENANT-ID", tenantName).header("X-FHIR-DSID", dataStoreId).get(Response.class);
+    }
+
+    public Response doPost(String path, String inputFormat, String inputSource, String resourceType, String url) throws FHIRGeneratorException, IOException {
+        return doPost(path, inputFormat, inputSource, resourceType, url, "default");
+    }
+
+    public Response doPost(String path, String inputFormat, String inputSource, String resourceType, String url, String provider)
+        throws FHIRGeneratorException, IOException {
+        WebTarget target = getWebTarget();
+        target = target.path(path);
+        if (DEBUG) {
+            System.out.println("URL -> " + target.getUri());
+        }
+        Parameters parameters = generateParameters(inputFormat, inputSource, resourceType, url, provider);
+        Entity<Parameters> entity = Entity.entity(parameters, FHIRMediaType.APPLICATION_FHIR_JSON);
+        return target.request(FHIRMediaType.APPLICATION_FHIR_JSON)
+                .header("X-FHIR-TENANT-ID", tenantName)
+                .header("X-FHIR-DSID", dataStoreId)
+                .header("X-FHIR-BULKDATA-PROVIDER", provider)
+                .header("X-FHIR-BULKDATA-PROVIDER-OUTCOME", provider)
+                .post(entity, Response.class);
+    }
+
+    public Response polling(String statusUrl) throws InterruptedException {
+        int status = 202;
+        int totalTime = 0;
+        Response response = null;
+        System.out.println("Started Checking");
+        while (Response.Status.ACCEPTED.getStatusCode() == status) {
+            response = doGet(statusUrl, FHIRMediaType.APPLICATION_FHIR_JSON);
+            // 202 accept means the request is still under processing
+            // 200 mean export is finished
+            status = response.getStatus();
+
+            assertTrue(status == Response.Status.OK.getStatusCode() || status == Response.Status.ACCEPTED.getStatusCode());
+
+            Thread.sleep(5000);
+            totalTime += 5000;
+            if (totalTime > 10 * 60 * 1000) {
+                fail("Too Long a Wait");
+            }
+        }
+        assertEquals(status, Response.Status.OK.getStatusCode());
+        System.out.println("Finished Checking");
+        return response;
+    }
+
+    /**
+     * {
+            "transactionTime": "2021-03-22T14:20:09.594Z",
+            "request": "$import",
+            "requiresAccessToken": false,
+            "output": [
+                {
+                    "type": "OperationOutcome",
+                    "url": "test-import.ndjson_oo_success.ndjson",
+                    "count": 4
+                }
+            ],
+            "error": [
+                {
+                    "type": "OperationOutcome",
+                    "url": "test-import.ndjson_oo_errors.ndjson",
+                    "count": 0
+                }
+            ]
+        }
```

Comment:
if you wrap this in a pre tag then it will maintain formatting when you look at it from eclipse/elsewhere